packages/node_modules/@webex/helper-image/src/index.js

```diff
@@ -49,18 +49,24 @@ export function readExifData(file, buf) {
       file &&
       (file.type === 'image/jpeg' || file.mimeType === 'image/jpeg')
     ) {
-      const {
-        Orientation,
-        ExifImageHeight,
-        ExifImageWidth
-      } = await parse(buf, {translateValues: false});
+      try {
+        const {
+          Orientation,
+          ExifImageHeight,
+          ExifImageWidth
+        } = await parse(buf, {translateValues: false});
 
-      file.orientation = Orientation;
-      file.exifWidth = ExifImageWidth;
-      file.exifHeight = ExifImageHeight;
+        file.orientation = Orientation;
+        file.exifWidth = ExifImageWidth;
+        file.exifHeight = ExifImageHeight;
 
-      if (file.image) {
-        file.image.orientation = Orientation;
+        if (file.image) {
+          file.image.orientation = Orientation;
+        }
+      }
+      catch (error) {
```

Comment:
The previous implementation checked if it error and just resolved the buffer if it did
```js
new ExifImage({image: buf}, (error, exifData) => {
        if (!error && exifData) {
          file.orientation = exifData.image.Orientation;
          file.exifWidth = exifData.exif.ExifImageWidth;
          file.exifHeight = exifData.exif.ExifImageHeight;
          if (file.image) {
            file.image.orientation = exifData.image.Orientation;
          }
        }
        resolve(buf);
      });
```

Reply:
I can also refactor it so that it's not a try catch and just a if against Orientation, ExifImageWidth, or ExifImageHeight existing before assigning

Reply:
I think a conditional would be better than throwing an error